app/models/order_status.rb

```diff
@@ -16,34 +16,29 @@ class OrderStatus < ActiveRecord::Base
   end
 
   scope :for_facility, ->(facility) { where(facility_id: [nil, facility.id]).order(:lft) }
-  scope :new_os, -> { where(name: "New").limit(1) }
-  scope :inprocess, -> { where(name: "In Process").limit(1) }
-  scope :canceled, -> { where(name: "Canceled").limit(1) }
-  scope :complete, -> { where(name: "Complete").limit(1) }
-  scope :reconciled, -> { where(name: "Reconciled").limit(1) }
 
   def self.new_status
-    new_os.first
+    find_by(name: "New")
   end
 
   def self.complete_status
```

Comment:
I like this. Though, now I almost want to go with `OrderStatus.complete` (without the somewhat redundant `_status`). Not completely necessary, though. Maybe leave both with one the alias for the other while we migrate?